learning/features/math/vector_math.rst

```diff
@@ -60,12 +60,12 @@ Vector Operations
 You can use either method (x and y coordinates or angle and magnitude) to
 refer to a vector, but for convenience programmers typically use the
 coordinate notation. For example, in Godot the origin is the top-left
-corner of the screen, so to place a 2D node 400 pixels to the right and
+corner of the screen, so to place a 2D node named ``Node2D`` 400 pixels to the right and
 300 pixels down, use the following code:
 
 ::
 
-    $Node2D.position = Vector2(400, 300)
+    $Node2D.position = Vector2(400, 300)  # "$" fetches the node by name from the tree
```

Comment:
I don't think this is especially helpful, especially after your addition above regarding the node name. If you feel like you need to explain "$" because you're thinking of an absolute beginner, then you can't use "tree" without explaining it either.

Reply:
I searched for the symbol `$` in the documentation within Godot and couldn't find it. I actually found it on the internet in an answer. That's why I thought I'd mention what it does.

As far as I'm concerned, I already know about it so I'm OK with what you suggest.

Reply:
The GDScript intro doc still has to be updated. Now that we're in beta, it will be a good time because there aren't likely to be any more changes to basic GDScript.